src/compiler/gcc.rs

```diff
@@ -331,6 +332,7 @@ pub fn compile<T>(creator: &T,
     cmd.arg("-c")
         .arg(&parsed_args.input)
         .arg("-o").arg(&out_file)
+        .args(&parsed_args.preprocessor_args)
         .args(&parsed_args.common_args)
```

Comment:
Not sure if this was a bug I inherited from the Python version or one I invented all on my own. We probably should just use the original arguments. I filed https://github.com/mozilla/sccache/issues/147 on doing that.